src/lib.rs

```diff
@@ -188,12 +190,16 @@ pub mod web {
             KeyupEvent,
             ClickEvent,
             DoubleClickEvent,
+            MouseDownEvent,
+            MouseUpEvent,
+            MouseMoveEvent,
```

Comment:
These were already exported in another PR, so please get rid of them.

Reply:
Yeah